<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="6" hostname="LAPTOP-9MGQ1891" failures="1" tests="3" name="Test" time="95.569" errors="0" timestamp="2022-07-12T13:17:44 IST">
  <testcase classname="com.Vtiger.TestCases.TC01_CreateOrg" name="createorg" time="17.585">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at com.Vtiger.TestCases.TC01_CreateOrg.createorg(TC01_CreateOrg.java:46)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- createorg -->
  <testcase classname="com.Vtiger.TestCases.TC02_CreateOrg_DD" name="createorg_dd" time="15.1"/>
  <testcase classname="com.Vtiger.TestCases.TC03_CreateContact_org" name="createcontact_org" time="34.582"/>
  <testcase classname="com.Vtiger.generic.BaseClass" name="@BeforeSuite connecttoDB" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite connecttoDB -->
  <testcase classname="com.Vtiger.generic.BaseClass" name="@BeforeSuite connecttoDB" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite connecttoDB -->
  <testcase classname="com.Vtiger.generic.BaseClass" name="@AfterSuite disConnectDB" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite disConnectDB -->
  <testcase classname="com.Vtiger.generic.BaseClass" name="@AfterSuite disConnectDB" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite disConnectDB -->
  <testcase classname="com.Vtiger.generic.BaseClass" name="@AfterTest closeBrowser" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterTest closeBrowser -->
  <testcase classname="com.Vtiger.generic.BaseClass" name="@AfterTest closeBrowser" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterTest closeBrowser -->
</testsuite> <!-- Test -->
